src/content/docs/en/guides/upgrade-to/v4.mdx

```diff
@@ -57,7 +57,7 @@ See [the changelog](https://github.com/withastro/astro/blob/main/packages/astro/
 
 Remove the following experimental flags from `astro.config.mjs`:
```

Comment:
How do you feel about something in this direction?

```suggestion
Remove the `devOverlay` experimental flag and move any `i18n` config to the top level in `astro.config.mjs`:
```

Reply:
Looks good!